dhash/database.py

```diff
@@ -102,6 +103,36 @@ def dump(self) -> dict:
         }
 
 
+class HashConfig(database.base):
+    """Stores config in format key:value"""
+
+    __tablename__ = "fun_dhash_config"
+
+    key = Column(String, primary_key=True)
+    value = Column(String)
```

Comment:
This completely ignores the fact that this would not work for multi-server environment. Each of the tables we use _should_ have `idx` and `guild_id` column.

As a second note, why is it key:value and not `allowed_urls` column? Better name should be `url_regex`, as it only holds string and not some kind of list/multi-value data.

Reply:
This is used only for protection against IP exposing for some paranoid people. So it's better there's only global option for two reasons:

1) If this is not set, dhash has no URL whitelist (it will scan any URL matching size and data type criteria).
2) If you restrict dhash to some domains / urls by regex, you want to do it for some reason, such as IP protection. And you want it to be for whole bot.

Also I hope this database object is only temporary since we were talking about some global solution in core for storing those lone values.

Reply:
Solved at Discord.